core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -686,14 +703,13 @@ object IO extends Serializable {
     final val AsyncIOEffect   = 6
     final val Redeem          = 7
     final val Fork            = 8
-    final val Race            = 9
-    final val Suspend         = 10
-    final val Uninterruptible = 11
-    final val Sleep           = 12
-    final val Supervise       = 13
-    final val Terminate       = 14
-    final val Supervisor      = 15
-    final val Ensuring        = 16
+    final val Suspend         = 9
+    final val Uninterruptible = 10
+    final val Sleep           = 11
+    final val Supervise       = 12
+    final val Terminate       = 13
+    final val Supervisor      = 14
+    final val Ensuring        = 15
```

Comment:
Nice to see these shrink! Also if you look at the PR, it's net negative! 🎉 🕺

Reply:
It says a lot about the expressiveness of ZIO 😍